Yeah tests fail too:

```
error: builder for '/nix/store/7bsiz0pd1b837nw67gdnirkfzjp36d1g-krep-0.3.3.drv' failed with exit code 2;
       last 19 log lines:
       > Running phase: unpackPhase
       > unpacking source archive /nix/store/bywhfcgs7zdndd98g4q0nys8zas4rzz9-source
       > source root is source
       > Running phase: patchPhase
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > build flags: SHELL=/nix/store/58br4vk3q5akf4g8lx0pqzfhn47k3j8d-bash-5.2p37/bin/bash CC=cc ENABLE_ARCH_DETECTION=0
       > cc -Wall -Wextra -O3 -std=c11 -pthread -D_GNU_SOURCE -D_DEFAULT_SOURCE -c krep.c -o krep.o
       > cc -Wall -Wextra -O3 -std=c11 -pthread -D_GNU_SOURCE -D_DEFAULT_SOURCE -o krep krep.o
       > Running phase: checkPhase
       > gcc -Wall -Wextra -O3 -std=c11 -pthread -D_GNU_SOURCE -D_DEFAULT_SOURCE -DTESTING -c test/test_krep.c -o test/test_krep.o
       > gcc -Wall -Wextra -O3 -std=c11 -pthread -D_GNU_SOURCE -D_DEFAULT_SOURCE -DTESTING -c test/test_regex.c -o test/test_regex.o
       > gcc -Wall -Wextra -O3 -std=c11 -pthread -D_GNU_SOURCE -D_DEFAULT_SOURCE -DTESTING -o test_krep test/test_krep.o test/test_regex.o krep.o
       > /nix/store/s40y31bdn82sj6daaxid1bn3p7la03lv-binutils-2.43.1/bin/ld: krep.o: in function `main':
       > krep.c:(.text.startup+0x6c0): multiple definition of `main'; test/test_krep.o:test_krep.c:(.text.startup+0x0): first defined here
       > collect2: error: ld returned 1 exit status
       > make: *** [Makefile:91: test] Error 1
       For full logs, run 'nix log /nix/store/7bsiz0pd1b837nw67gdnirkfzjp36d1g-krep-0.3.3.drv'.
```

--- a/pkgs/by-name/kr/krep/package.nix
+++ b/pkgs/by-name/kr/krep/package.nix
@@ -1,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krep";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "davidesantangelo";
+    repo = "krep";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-kAsOAcEFjfxlJs6fJvB0viCMxGFCG1BUs9qPgGMvBpM=";
+  };
+
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+    "ENABLE_ARCH_DETECTION=0"
+  ];
+
+  installFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
+  doCheck = true;
+  doInstallCheck = true;
+  nativeInstallCheck = [ versionCheckHook ];
+
+  meta = {
+    description = "Blazingly fast string search utility designed for performance-critical applications";
+    homepage = "https://github.com/davidesantangelo/krep";
+    changelog = "https://github.com/davidesantangelo/krep/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.bsd2;
+    maintainers = with lib.maintainers; [
+      codebam
+    ];
+    platforms = lib.platforms.unix;
+  };
+})

